.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ commands:
       - run:
           name: Output BUILD_VERSION env var based on recent git tag and type of build
           command: |
-            BUILD_VERSION=$(git describe --tags --match 'v*.*.*' --exclude '*feature');
```

Comment:
👁️‍🗨️ Accepting the `-feature` suffix matches the description of collecting the build version, and matches the tag that `goreleaser` expects:

https://github.com/slackapi/slack-cli/blob/12daa9ae4c941e2767a46553baab64a6838fd41a/.circleci/config.yml#L23